### PR TITLE
EC2 now supports virtualised instances via Libvirt

### PR DESCRIPTION
**Pull request by viren-nadkarni**
https://localstack-docs-preview-pr-1132.surge.sh/user-guide/aws/ec2/

To be merged with the companion PR.

### CODE REVIEW

File: content/en/user-guide/aws/ec2/index.md [hunk 1]
```diff
@@ -212,12 +222,12 @@ $ docker tag ubuntu:focal localstack-ec2/ubuntu-focal-ami:ami-000001
 The above example will make LocalStack treat the `ubuntu:focal` Docker image as an AMI with name `ubuntu-focal-ami` and ID `ami-000001`.
 
 At startup, LocalStack downloads the following AMIs that can be used to launch Dockerized instances.
-This behaviour can be controlled using the `EC2_DOWNLOAD_DEFAULT_IMAGES` configuration variable.
+- Ubuntu 20.04 `ami-ff0fea8310f3` (deprecated and marked for removal in the next major release)
```
Comment (macnev2013) [inline content/en/user-guide/aws/ec2/index.md:225]: Maybe we can add a release number here instead of `next major release`

**Review by macnev2013 — APPROVED** [review]
Just a minor nit else LGTM 👍🏻

**Review by HarshCasper — APPROVED** [review]

File: content/en/user-guide/aws/ec2/index.md (hunk 1 above)
Comment (viren-nadkarni) [reply content/en/user-guide/aws/ec2/index.md:225]: It's intentional -- not to make committments and to give us some flexibility :slightly_smiling_face:

**Review by macnev2013 — APPROVED** [review]
LGTM 👍🏻

**Pull request merged**